package.json

```diff
@@ -29,9 +29,16 @@
   },
   "devDependencies": {
     "husky": "^4.3.6",
+    "lint-staged": "^10.5.3",
```

Comment:
Can you post a demo of how this is working on our project? also, what will happen in case of non-auto fixable rules?

Reply:
Demo is ready!